### DISCUSSION TITLE
Expand content area for a single section

### DISCUSSION THREAD

**Discussion by Ly0n** [squidfunk/mkdocs-material#4188] [Ideas]
I've been trying to integrate an AirTable iframe cleanly for some time now. For working with the table it would be desirable to extend the content area for a single section to the maximum size. Here how it is integrated at the moment: https://opensustain.tech/discover/ 
However, with template customization I am only able to do this for the entire site:

```
.md-grid {
  max-width: initial;
}
```
Can this customization only be done for one content tab?

**Comment by squidfunk** [comment]
Yes, just add your customizations to the Markdown document behind the specific page with

``` html
<style>
  .md-grid {
    max-width: initial;
  }
</style>
```

**Reply by Ly0n** [reply]
Thank you for the lightning reply. The solution you suggested works: 
https://opensustain.tech/discover/

The top navigation bar is also extended but this should work for most users! I was also able to remove the title of the page like you already solved here:
https://github.com/squidfunk/mkdocs-material/issues/2163 

Thank you so much for your support. I hope to get soon funding for the projects so that we can give you again some donations.
